.github/pull_request_template.md

```diff
@@ -1,10 +1,18 @@
+<!-- 
```

Comment:
I think the downside of leaving these as comments is that it's not seen by default when you create a PR. Not sure how to fix that, but otherwise it looks good.

Reply:
I think the comments are visible in the edit mode, but maybe there are settings how edit mode looks? That's what I see when I create new PR:
![image](https://github.com/user-attachments/assets/13b07eed-d74f-4f27-af94-fcc51d1537ed)